grammar.y

```diff
@@ -35,6 +35,15 @@ var keywordsNotAllowed = map[string]struct{}{
   "UNION": {},
 }
 
+func isRowID(column Identifier) bool {
+	lowered := strings.ToLower(string(column))
+	if lowered == "rowid" || lowered == "_rowid_ " || lowered == "oid" {
```

Comment:
Nit: there's a particular method in `strings` for case insensitive comparisons, [strings.EqualFold](https://pkg.go.dev/strings#EqualFold). It probably would skip doing the copy in L39. Mentioning just if you want to give it a shot, but this is fine too.